.github/ISSUE_TEMPLATE/03-testgap.yml

```diff
@@ -0,0 +1,60 @@
+name: Report a Test Gap

```

Comment:
Should we use the "sonic-net/sonic-mgmt" repo to track test gaps?

Reply:
Thanks for the comment. I see that we don't use Github projects for tracking. So I have removed it.